crates/next-custom-transforms/tests/full/example/output.js

```diff
@@ -4,5 +4,5 @@ import r from 'other';
 e(r, 1)[0];
 export var __N_SSG = !0;
 export default function t() {
-    return /*#__PURE__*/ React.createElement("div", null);
+    return React.createElement("div", null);
```

Comment:
Is that expected? Probably not?

Reply:
It's the effect of https://github.com/swc-project/swc/pull/9775 and it's intentional, as this test case is about `full` processing. SWC nows drops `PURE` comments minifiying the final bundle